### PR TITLE
Framework(Baseyii.php)  -----> fixed typos and improved doc

### PR DESCRIPTION
**Pull request by Biki-das**

### CODE REVIEW

File: framework/BaseYii.php [hunk 1]
```diff
@@ -201,9 +201,9 @@ public static function getRootAlias($alias)
      * See the [guide article on aliases](guide:concept-aliases) for more information.
      *
      * @param string $alias the alias name (e.g. "@yii"). It must start with a '@' character.
-     * It may contain the forward slash '/' which serves as boundary character when performing
+     * It may contain the forward-slash '/' which serves as a boundary character when performing
      * alias translation by [[getAlias()]].
-     * @param string $path the path corresponding to the alias. If this is null, the alias will
+     * @param string $path The path corresponding to the alias. If this is null, the alias will
```
Comment (bizley) [inline framework/BaseYii.php:206]: Lowercase `the` is proper.

File: framework/BaseYii.php [hunk 2]
```diff
@@ -549,7 +549,7 @@ public static function t($category, $message, $params = [], $language = null)
     /**
      * Configures an object with the initial property values.
      * @param object $object the object to be configured
-     * @param array $properties the property initial values given in terms of name-value pairs.
+     * @param array $properties the property initial values are given in terms of name-value pairs.
```
Comment (bizley) [inline framework/BaseYii.php:552]: `are` is not right here.

File: framework/BaseYii.php [hunk 3]
```diff
@@ -566,7 +566,7 @@ public static function configure($object, $properties)
      * This method is provided such that we can get the public member variables of an object.
      * It is different from "get_object_vars()" because the latter will return private
      * and protected variables if it is called within the object itself.
-     * @param object $object the object to be handled
+     * @param object $object The object to be handled
```
Comment (bizley) [inline framework/BaseYii.php:569]: Lowercase `the` is proper.

**Comment by Biki-das** [comment]
@bizley , thanks for the feeback and review

**Comment by Biki-das** [comment]
@bizley  will it need more changes or its fine?

**Comment by bizley** [comment]
All good. Thanks!

**Comment by Biki-das** [comment]
> All good. Thanks!

always happy to help and contribute :-)!

**Pull request merged**